lib/components/auth/digest.ts

```diff
@@ -1,6 +1,6 @@
 // https://tools.ietf.org/html/rfc2617#section-3.2.1
 
-import MD5 from 'md5.js'
```

Comment:
We use import statements in Typescript, and the compiler should take care of this. Was this necessary to get it working on Node ?!

Reply:
Yes. 
But to be honest, I don't understand whats going wrong exactly

**Orignal import statement brings following error:**  
`/media_stream_library/lib/components/auth/digest.ts:24  
    this.ha1Base = new MD5()  
                   ^  
TypeError: md5_js_1.default is not a constructor  
    at new DigestAuth `  

  

**This would be my natural import: (works in node)**
```typescript
import * as MD5 from 'md5.js'
```
But this results in error in the test:
``` Summary of all failing tests
 FAIL  lib/components/auth/digest.test.ts
   digest challenge › generates the correct authentication header

    TypeError: MD5 is not a constructor



      at new DigestAuth (lib/components/auth/digest.ts:2195:20)
      at Object.it (lib/components/auth/digest.test.ts:9:20)  
```

Reply:
I‘ll check this asap, but currently I have no access to my PC.

Reply:
> After running `yarn build`, I've added the following line to `dist/cjs/components/auth/digest.js`:
> 
> ```
> const a = new md5_js_1.default()
> ```
> 
> and then ran:
> 
> ```
> node dist/cjs/components/auth/digest.js
> ```
> 
> which seems to work fine.
> Could you try the same thing and see what you get?
> If we can't figure it out, it would be best if you add an issue with your problem, and a description of how we might replicate it.
> 
> (I'm using Node JS version 12)

Thanks for your support.
I could fix this issue, by using the dist version also (before I was referencing the lib directly), which is absolutely fine in my use case.